internal/rpc/apiproxy_test.go

```diff
@@ -275,8 +277,8 @@ func TestProxySocketsAdminFacade(t *testing.T) {
 			go func() {
 				defer wg.Done()
 				err = rpc.ProxySockets(ctx, helpers)
-				if test.expectConnectTofail {
-					c.Assert(err, qt.ErrorMatches, "failed to connect to controller: controller connection error")
+				if test.expectProxyError {
```

Comment:
could we simplify this and do 
```
if test.proxyErrorMessage != ""
```
and lose the bool?

Reply:
Yes, done.